### PR TITLE
Use npmjs package version in examples + new examples and checks for imports

### PR DESCRIPTION
**Pull request by elboletaire**
What's done in this PR:
- Two new examples (mostly a copy from the `esm` one): a typescript and a commonjs one.
- Examples now use a static version of the published package in npmjs, rather than using `file:../..`
- New action to check if imports are working well (tested in every example, except for CRA)
- Added a new really simple file to each example, named `check-import`, where only a proper import of the SDK is done, and an exception is thrown in case it's not properly imported.

You can see a successful run of the new github action here: https://github.com/elboletaire/vocdoni-sdk/actions/runs/4063850518

### CODE REVIEW

**Comment by elboletaire** [comment]
>     * Naming of each project in `package.json` (I see the same in each example)
Are you sure? I bet I changed them in a forced push a few hours ago.
>     * Missing `@ethersproject/wallet` dependency
Hhmmm.. true, altho I don't get how are they working locally tbh without the dependency installed. I'll have to take a look
>     * Why the `EnvOptions.DEV is not "dev"` message?
Because is what it actually checks... I could have set a message like "import didn't work", but it would be appropriate either, since it may not be actually directly related to the import... so I decided to just show an error noting what's going on there. Note that this could be an empty throw... it's just there to properly check the imports working (rather than just importing and doing nothing, which could also work btw).

**Comment by elboletaire** [comment]
Updated with the following changes:
- bundle checks moved to `test/bundle`
- replaced the typescript import test with one for the UMD package, since we're testing different outputs here, and the technology being used doesn't matter that much (the typescript example has been maintained tho)
- bundle test message updated to avoid confusion
- added `@ethersproject` dependencies to examples that are directly using them

You can check a properly ran gh action here: https://github.com/elboletaire/vocdoni-sdk/actions/runs/4064042426

**Review by marcvelmer — APPROVED** [review]
LGTM

Will merge later!

File: test/bundle/umd/readme.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Vocdoni SDK ESM Typescript example
```
Comment (elboletaire) [inline test/bundle/umd/readme.md:1]: This readme should be removed

**Pull request merged**